src/machine/machine_rp2.go

```diff
@@ -1,15 +1,44 @@
-//go:build rp2040
+//go:build rp2040 || rp2350
 
 package machine
 
 import (
 	"device/rp"
+	"runtime/interrupt"
 	"runtime/volatile"
 	"unsafe"
 )
 
 const deviceName = rp.Device
 
+const (
+	// Number of spin locks available
+	_NUMSPINLOCKS = 32
+	// Number of interrupt handlers available
+	_NUMIRQ               = 32
+	_PICO_SPINLOCK_ID_IRQ = 9
+)
+
+// UART on the RP2040
```

Comment:
We should update comment:
```suggestion
// UART on the RP2040/RP2350
```